src/main/kotlin/edu/agh/susgame/dto/SocketMessage.kt

```diff
@@ -0,0 +1,17 @@
+package edu.agh.susgame.dto
+
+import kotlinx.serialization.Serializable
+
+
+@Serializable
+sealed class SocketMessage {
+    // TODO CDM-72 Pretty much everything
+    @Serializable
+    data object GameState : SocketMessage()
+
+    @Serializable
+    data class SimpleMessage(
```

Comment:
To @Uiskam, dodaj proszę w takim razie to `SimpleMessage` zarówno do `ServerSocketMessage`, jak i `ClientSocketMessage` z jakimś komentarzem typu:
```
/**
  * Will be deleted in the future
  */
```

Wersja w `ClientSocketMessage` chyba może być pozbawiona nicku, bo serwer to sobie sam ogarnie. Możesz po tym wywalić to całe `SocketMessage.kt`

Reply:
jak to działa i miało być na prezentację to można zostawić po prostu tak jak jest